zeppelin-web/src/app/helium/helium.controller.js

```diff
@@ -47,6 +47,7 @@
         }
       }
       $scope.defaultVersions = defaultVersions;
+      console.log(_.size(defaultVersions));
```

Comment:
Right I should've removed it before. It was my bad. Removed it in baf4a7773373bde4dac1cc71a07de3c5441615ab :)